templates/style/vendored.scss

```diff
@@ -1,2 +1,2 @@
-@import "pure-min", "grids-responsive-min";
+@import "pure-min.css", "grids-responsive-min.css";
```

Comment:
This doesn't actually work, it makes it emit a CSS `@import` instead of actually inlining the files.